doc/man1/req.pod

```diff
@@ -242,7 +242,6 @@ days to certify the certificate for. The default is 30 days.
 
 serial number to use when outputting a self signed certificate. This
 may be specified as a decimal value or a hex value if preceded by B<0x>.
-It is possible to use negative serial numbers but this is not recommended.
```

Comment:
Ok, fair enough. So what used to work still does (now). I also agree about the morality and decency of doing this!! It's just a question of whether we mention this or not. I have a tendency to prefer to document the behaviour (no matter how repugnant) than not. But I'm not overly worried about it - so I'll leave it up to you to decide.